vue/components/ui/molecules/avatar-list/avatar-list.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+    <div class="avatar-list">
+        <div
+            class="avatar-container"
+            v-bind:class="classes"
+            v-bind:style="{ 'z-index': index }"
+            v-for="(avatar, index) in avatarList"
+            v-bind:key="index"
+        >
+            <div v-if="index == maxElems">
+                + {{ avatarList.length - maxElems }}
+            </div>
+            <avatar v-bind="{ size: size, ...avatar }" v-else-if="index < maxElems" />
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+.avatar-list {
+    display: flex;
+}
+
+.avatar-list > .avatar-container > * {
+    border: 2px solid #ffffff;
+}
+
+.avatar-list > .avatar-container:not(:first-child) {
+    transition: margin 0.3s;
+}
+
+.avatar-list > .avatar-container:not(:first-child).very-tiny {
+    margin-left: -10px;

```

Comment:
I believe our standard doesn't allow for negative margins

Reply:
In this case, the negative margins is to force the overlap. There was another solution previously in place, using absolute and relative positions. Unfortunately, after that solution was implemented I was told that the avatars also needed to expand when hovered, and that presented problems. When using absolute positions, the parent component has no width and height, so the hover presented problems because when expanding the user could hover over "nothing", forcing the collapse.

This solution doesn't use absolute and relative positioning, so the only way to enforce overlap is through negative margins, that turn positive when hovered.

Reply:
I agree with you that it is a simpler solution. I'm not sure about how strict we are with this standard of if it even is one at all so let's ignore it for the time being at least